src/include/daos/debug.h

```diff
@@ -59,7 +59,11 @@
 	ACTION(duns,      duns,      arg)	\
 	ACTION(drpc,      drpc,      arg)	\
 	ACTION(security,  security,  arg)	\
-	ACTION(dtx,       dtx,       arg)
+	ACTION(dtx,       dtx,       arg)	\
+	ACTION(cn,        cnss,      arg)	\
+	ACTION(cli,       cli,       arg)	\
+	ACTION(ctrl,      ctrlfs,    arg)	\
```

Comment:
Do we need the controlfs? If so, why?

Reply:
The controlfs has been used for three things

- Exporting projection-wide information to the interception library
- To facilitate clean shutdown of fuse by the application
- Exporting of statistics about what I/O operations are being performed

Given we previously needed to support multiple projections per node, and in unified namespace we don't then perhaps the 1st of these could be better supported by a per file ioctl() now, and we could come up with other ways of achieving the others.  It would allow us to remove a lot of complexity.

Reply:
in cppr we used it to interact with the service as well.   It provides a few APIs for registering variables that are either read only constants or interactive (e.g. write a value, get back another value).   Also, we used it to assign a unique id to each cppr client and to track the death of the client application.

We used it in our testing as well.   We had a simple query mechanism where we wrote a path to a file and got the tree root for the cooperative cache back on reading.  This greatly simplified some of our testing as we could let cppr give us information.

Aside from shutdown, we also used it for debugging where we could trigger a state dump to the CPPR log or modify the log mask at runtime, etc.

It may be less useful for DAOS use case since dfs is just a client.   But it was fairly useful for CPPR.